sdcm/nemesis.py

```diff
@@ -1587,6 +1587,82 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_remove_node_then_add_node(self):
+        """
+        https://docs.scylladb.com/operating-scylla/procedures/cluster-management/remove_node/
+
+        1.Terminate node
+        2.Run full repair
+        3.Nodetool removenode
+        4.Add new node
+        """
+
+        self._set_current_disruption('TerminateAndRemoveNodeMonkey')
+        node_to_remove = self.target_node
+        up_normal_nodes = self.cluster.get_nodes_up_and_normal(verification_node=node_to_remove)
+        # node_to_remove must be different than node
+        node = random.choice([n for n in self.cluster.nodes if n is not node_to_remove])
+
+        # node_to_remove must not be the only seed in cluster
+        num_of_seed_nodes = len(self.cluster.seed_nodes)
+        if node_to_remove.is_seed and num_of_seed_nodes < 2:
+            raise UnsupportedNemesis("Removing the only seed node is not yet supported")
```

Comment:
We have another parameter that is called "fliter_seed" and it determines if we run nemesis on seed or not.
I think we should take it into consideration also here.

In addition (in a followup PR), change this parameter to "false" in 2-3 longevities and increase the number of seeds in these longevities to 2 and 3.

Reply:
@nizarh11 please fix the usage of this param (filter_seed) in a following PR.